docs/publishing.md

```diff
@@ -1,37 +1,46 @@
 # Publishing a new version of `@mdn/browser-compat-data`
 
-[Project owners](./GOVERNANCE.md#owners) publish new releases of [`@mdn/browser-compat-data`](https://www.npmjs.com/package/@mdn/browser-compat-data) on npm, usually every Tuesday and Friday.
-MDN staff then deploy the package to the MDN site, usually every Thursday (MDN never deploys to production on Fridays).
+[Project owners](./GOVERNANCE.md#owners) publish new releases of the [`@mdn/browser-compat-data`](https://www.npmjs.com/package/@mdn/browser-compat-data) NPM package, usually every Tuesday and Friday.
 
-## Before you begin
+The release process is mostly automated using two GitHub workflows:
 
-Any project owner (or release designee) can complete the following steps to publish a new version.
-
-The steps in this process assume:
-
-- `NPM_TOKEN` is set in the repository secrets. If the token is invalidated or unset, a member of the `@mdn` organization on npm must [create a new token](https://docs.npmjs.com/creating-and-viewing-authentication-tokens) and [add it to the repository's secrets](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets-for-a-repository).
-- You have cloned the mdn/browser-compat-data repository and you have the latest `main` branch checked out.
-- You have permissions to merge pull requests and publish releases.
-- You have [the `gh` CLI](https://cli.github.com/) installed.
-- You have [`jq`](https://stedolan.github.io/jq/) installed.
+1. The [`release-pr` workflow](https://github.com/mdn/browser-compat-data/blob/main/.github/workflows/release-pr.yml) manages a pull request draft that bumps the `package.json` version and updates the release notes.
+2. The [`release` workflow](https://github.com/mdn/browser-compat-data/blob/main/.github/workflows/release.yml) creates the GitHub release and tag, and publishes the release on NPM.
 
 ## Performing a release
 
-Releasing a new version of BCD is a two-step process: first, the release script is run to bump version number, generate the changelog, and create a pull request; then, once the pull request is merged, a release on GitHub is performed.
+A release is usually performed by merging the pull request managed by the `release-pr` workflow:
```

Comment:
"Usually"? When is it not? Or if you're not interested in covering the exceptional cases, drop "usually."